### PR TITLE
Fixed a problem of the script to register st2 configuration to Zabbix and added an integration test for it.

### LINKED ISSUE #28: user.addmedia is deprecated in Zabbix 4.0
When running register_st2_config_to_zabbix.py against a Zabbix 4.x installation, the script fails trying to add a user media.  
Per Zabbix 3.4 documentation https://www.zabbix.com/documentation/3.4/manual/api/reference/user/addmedia
"This method is deprecated and will be removed in the future. Please use user.update instead."

Error when running the script
pyzabbix.api.ZabbixAPIException: ('Error -32602: Invalid params., Incorrect method "user.addmedia". while sending ......

### PR DESCRIPTION
**Pull request by userlocalhost**
This fixed #28. Furthermore, this added an integration test of running `tools/register_st2_config_to_zabbix.py` with Zabbix v3.2 and v4.0.

### CODE REVIEW

**Comment by userlocalhost** [comment]
Sorry, I forgot to amend a version in `pack.yaml` and write a changelog. Please wait to merge it.

**Comment by userlocalhost** [comment]
> to amend a version in pack.yaml and write a changelog.

It's done.

**Comment by armab** [comment]
Integration test looks very solid :+1: 

I'm now wondering why CircleCI doesn't show the build status for the commited PR changes. Guessing it's something related to https://github.com/StackStorm-Exchange/stackstorm-zabbix/pull/29/checks, - I've seen something similar when enabling new Github "checks" feature that stopped the builds to show up. Will try something to fix it.

**Comment by userlocalhost** [comment]
> I'm now wondering why CircleCI doesn't show the build status for the commited PR changes.

Sorry, It's my fault. I set configuration of CircleCI to follow forked repository (`userlocalhost/stackstorm-zabbix`). When forked repository was followed, the test of PR seems not to be run.

After I unregistered to following forked repository in CircleCI, then it works well.

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  task :all     => targets
+  task :default => :all
+
+  targets.each do |target|
+    original_target = target == "_default" ? target[1..-1] : target
+    desc "Run serverspec tests to #{original_target}"
+    RSpec::Core::RakeTask.new(target.to_sym) do |t|
+      ENV['TARGET_HOST'] = original_target
+      t.pattern = "spec/#{original_target}/*_spec.rb"
+    end
```
Comment (armab) [inline Rakefile]: I'd suggest to use extended `documentation` format that'll show more detailed tests output and take full advantage of RSpec `it` vs `describe`.

See https://relishapp.com/rspec/rspec-core/v/2-5/docs/command-line/format-option#documentation-format

Comment (armab) [reply Rakefile]: Currently it shows the tests progress like the following:
```
....
```

Comment (userlocalhost) [reply Rakefile]: I corrected configuration of CircleCI (`.circleci/config.yml`) to show the test progress in detail.
スクリーンショット 2019-03-27 10 57 44
(c.f. https://circleci.com/gh/StackStorm-Exchange/stackstorm-zabbix/96)

**Pull request merged**